docs/source/models/gbd2017/causes/lower_respiratory_infections/index.rst

```diff
@@ -17,12 +17,29 @@ syncytial virus (*RCV*). LRI can be caused by multiple pathogens and the pathoge
 Pneumococcal pneumonia is the largest cause of LRI
 mortality. [Wikipedia]_, [GBD-2017-YLD-Capstone-Appendix-1]_
 
+Transmission of LRI may occur via several pathways, including direct physical contact,
+fomites, direct droplet spread, and suspended small particles. Intermingling of
+large numbers of people can facilitate transmission of respiratory pathogens. [CDC]_
+
+Malnutrition was identified as a leading risk factor for lower respiratory infection
+mortality among children younger than 5 years and, together with air pollution (both household and ambient)
+and increased antibiotic use, was identified as a focus for targeted intervention measures. [Lancet]_
+
+.. todo::
+
+   Describe more about deaths and complications due to LRI.
+   Talk about current vaccination against influenza and pneumonia.
+   https://apps.who.int/iris/bitstream/handle/10665/241904/WER8714_129-144.PDFp
+
 Modeling LRI in GBD 2017
 ------------------------
 
 The GBD 2017 defined the time to recovery for LRI as an average of 10 days(*5-15 days*),
```

Comment:
```suggestion
The GBD 2017 defined the time to recovery for LRI as an average of 10 days (*5-15 days*),
```
Missing a space. You're also missing a space after a comma in the first paragraph: "(Hib),influenza" should be "(Hib), influenza"